site2016/templates/site2016/membro.html

```diff
@@ -128,6 +128,17 @@ <h4>Tempo gasto: {{ atividade.horas }}h{{ atividade.minutos }}min</h4>
                 </div>
             </div>
             {% endfor %}
+
+            <div class="ui divider"></div>
+            <div class="row centered">
+                <div class="ui grid" style="max-width: 40%;">
```

Comment:
```suggestion
                <div class="ui grid middle aligned" style="max-width: 40%;">
```

Resolve o problema de ficar verticalmente desalinhado